tests/Entities/ProjectEntityTest.php

```diff
@@ -1,28 +0,0 @@
-<?php
```

Comment:
why has this been deleted??

Reply:
We deleted that because we didn't needed any unity test for the Project Entity

Reply:
there are still 2 getters in the entity though. you need to mark those as skipped because they require integration tests